packages/gestalt/src/Flex.js

```diff
@@ -15,8 +15,22 @@ type Props = {|
   alignContent?: 'start' | 'end' | 'center' | 'between' | 'around' | 'evenly' | 'stretch',
   /**
    * Defines the default behaviour for how flex items are laid out along the cross axis on the current line. Think of it as the `justify-content` version for the cross axis (perpendicular to the main axis).
+   *
+   * Also available in responsive sizes: `smAlignItems`, `mdAlignItems`, `lgAlignItems`. See the [Screen size page](https://gestalt.pinterest.systems/foundations/screen_sizes#Web-(px)) to learn more about viewport breakpoints in web.
    */
   alignItems?: 'start' | 'end' | 'center' | 'baseline' | 'stretch',
+  /**
```

Comment:
These props aren't automatically filtered out of the generated props table. You'll need to add them to [Box's ignored props](https://github.com/pinterest/gestalt/blob/master/docs/pages/web/box.js#L30), and set up something similar for Flex.

<img width="1004" alt="Screen Shot 2022-11-28 at 12 15 47 PM" src="https://user-images.githubusercontent.com/12059539/204372670-453ff269-fc7a-4285-9ffc-0af7737de676.png">